templates/sample-proj.yaml

```diff
@@ -29,7 +29,7 @@ items:
           deploymentconfig: sdp-example-proj
       spec:
         containers:
-        - image: "docker-registry.default.svc:5000/sample-app/sdp-example-proj:{{- include "determine_image_tag" .}}"
+        - image: "docker-registry.default.svc:5000/sample-app/sdp-example-proj:{{- include "determine_image_tag" (dict "root" .Values "repoName" "sdp_example_proj") }}"
```

Comment:
What's your opinion of putting the dict definition into its own line, and/or just passing the repos rather than the whole context
```
{{- $data := dict "Repos" .Values.global.repos "RepoName" "sdp-example-proj" }}
- image: "docker-registry.default.svc:5000/sample-app/sdp-example-proj:{{- include "determine_image_tag" $data}}"
```

Reply:
don't think it matters, we can document both ways.  at this point we've proven this approach is valid.  chart authors can choose the syntax they prefer.